WebSharper.Templates/templates/ClientServer-FSharp/Main.html

```diff
@@ -1,74 +1,98 @@
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                </button>
-                <a class="navbar-brand" href="#">Your App</a>
-            </div>
-            <div id="navbar" class="navbar-collapse collapse">
-                <ul class="nav navbar-nav" ws-hole="MenuBar"></ul>
-                <ul class="nav navbar-nav navbar-right">
-                    <li><a href="https://websharper.com">websharper.com</a></li>
-                </ul>
-            </div><!--/.nav-collapse -->
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <div class="container-fluid">
+        <div class="navbar-header">
+          <a class="navbar-brand" href="#">Your App</a>
         </div>
+        <div id="navbar" class="collapse navbar-collapse">
+          <ul class="navbar-nav me-auto mb-2 mb-lg-0" ws-hole="MenuBar"></ul>
+          <ul class="nav navbar-nav navbar-right">
+            <li>
+              <a href="https://websharper.com" class="nav-link">WebSharper</a>
+            </li>
+          </ul>
+        </div>
+      </div>
     </nav>
-    <div class="container">
-        <div ws-replace="Body">
-            <div ws-template="MainForm">
-                <input ws-var="TextToReverse" />
-                <button ws-onclick="OnSend">Send</button>
-                <hr />
-                <h4 class="text-muted">The server responded:</h4>
-                <div class="jumbotron"><h1>${Reversed}</h1></div>
-            </div>
+    <div class="container m-2 p-2">
+      <div ws-replace="Body">
+        <div ws-template="MainForm">
+          <hr class="w-50" />
+          <div class="input-group w-50 my-3">
+            <input
+              type="text"
+              class="form-control"
+              aria-label="message"
+              ws-var="TextToReverse"
+            />
+            <button class="btn btn-outline-primary" ws-onclick="OnSend">
+              Send
+            </button>
+          </div>
+          <div class="alert alert-dark w-50" style="word-wrap: break-word">
+            Server response: ${Reversed}
+          </div>
         </div>
+      </div>
     </div>
     <footer class="footer">
-        <div class="container">
-            <p class="text-muted">
-                For an enhanced template that provides automatic GitHub deployment to Azure, fork from <a href="https://github.com/intellifactory/ClientServer.Azure">GitHub</a>, or
-                read more <a href="https://websharper.com/blog-entry/4368/deploying-websharper-apps-to-azure-via-github">here</a>.
-            </p>
-        </div>
+      <div class="container">
+        <p class="text-muted text-center">
+          For an enhanced template that provides automatic GitHub deployment to
+          Azure, fork from
+          <a href="https://github.com/intellifactory/ClientServer.Azure"
+            >GitHub</a
```

Comment:
See comment at the same line for the C# client-server project.

Reply:
done